frontend/src/GetDecodedToken.js

```diff
@@ -0,0 +1,11 @@
+import { useJwt } from "react-jwt";
+import useToken from './hooks/useToken'
+
+function GetDecodedToken() {
```

Comment:
Ezt mindenképpen useDecodedToken-nek nevezzük, mert ez custom hook, ami más hookokat is használ.